terraform-static-analysis/action.yml

```diff
@@ -61,6 +61,10 @@ inputs:
   trivy_skip_dir:
     description: "Directories that get skipped for trivy e.g. test"
     required: false
+  main_branch_name:
+    description: "The name of the main branch e.g. 'main' or 'master', defaults to 'main'"
```

Comment:
`master` should not be used. The statement implies it is an available option when it shouldn't be. MoJ GitHub Standard is `main` so we should not be suggesting any alternatives.

Reply:
So the options will be based on how the clients repository is set up and which branch they want to compare against - ideally, they're using `main` as you've suggested, although they may be using a different name such as `master`, `default`, `this-branch-here-is-the-one-i-want-to-compare against` etc. 🤹 

So the suggestions are more for context of what their values might be i.e. if they're not using `main` currently, they're probably using `master`. Although we default to `main` to nudge people in that direction 👀 

But happy to completely remove if it's an issue!

Reply:
It should be removed. It goes against MoJ repository standards and MoJ inclusive language policy. 

I appreciate that teams may use other alternatives to `main`, which is acceptable, however use of `master`, specifically, is unacceptable.

Reply:
Gone 🔥